backend/controllers/prisonerProfile/prisonerCsraHistory.js

```diff
@@ -0,0 +1,58 @@
+const { putLastNameFirst, formatName, formatTimestampToDate, sortByDateTime } = require('../../utils')
+
+const csraOptions = [
+  { value: 'STANDARD', text: 'Standard' },
+  { value: 'HI', text: 'High' },
+  { value: 'PEND', text: 'Pending' },
+]
+
+module.exports = ({ prisonApi }) => async (req, res) => {
+  const { offenderNo } = req.params
+  const { csra, location } = req.query
```

Comment:
We need to start handling missing query parameters like these. Need to either redirect or respond with a something went wrong page and a HTTP 400 status.